actionsystemtest/lib/action_system_test/test_helpers/assertions.rb

```diff
@@ -0,0 +1,33 @@
+module ActionSystemTest
+  module TestHelpers
+    # Assertions for system testing that aren't included by default in Capybara.
+    # These are assertions that are useful specifically for Rails applications.
+    module Assertions
+      # Asserts that all of the provided selectors are present on the given page.
+      # Using the selector, +:avatar+ for example, asserts that the selector is
+      # present with the items passed in. If options are provided, the assertion
+      # will check that each item is present with those options as well.
+      #
+      #   assert_all_of_selectors(:avatar, 'Eileen', 'Jeremy')
+      #   assert_all_of_selectors(:avatar, 'Eileen', 'Jeremy', visible: all)
+      def assert_all_of_selectors(selector, *items, **options)
```

Comment:
I think these helpers are quite useful, but I'd rather see them moved upstream to Capybara itself, otherwise we're creating a "Rails-dialect" of Capybara, which I think would be much harder for newer users to grasp.